code/game/machinery/medical_kiosk.dm

```diff
@@ -231,13 +231,13 @@
 	var/addict_list = list()
 	var/hallucination_status = "Patient is not hallucinating."
 
-	if(altPatient.reagents.reagent_list.len)	//Chemical Analysis details.
+	if(LAZYLEN(altPatient.reagents.reagent_list))	//Chemical Analysis details.
```

Comment:
Is reagent_list lazy too? (Check and see if things that add to it use the LAZY macros, and see if you can find where it gets initialized) 

If not, length() (which is what LAZYLEN is under the hood) and .len are about the same, except length() is something like one operation faster, and .len runtimes when used on a null value. Use whichever you'd like.

Reply:
reagent_list seems to be intialised in holder.dm as a new/list(). From what I can tell, add_reagent and add_reagent_list do not add things using the LAZY macros.

Reply:
reagent_list shouldn't be initialised as a null, so I think I'll revert it to the len, but keep a ? for reagents since that is initialised as a null.